chameleon/tasks.py

```diff
@@ -131,6 +135,43 @@ def _migrate_project(region, username=None, charge_code=None, access_token=None)
         glance.image_members.update(image.id, ks_legacy_project.id, 'accepted')
         progress += ((1.0 - init_progress) / num_images)
 
+
+    # Migrate cinder volumes if kvm region
+    if region == 'KVM@TACC':
+        unscoped_session = unscoped_user_session(
+            region, access_token=access_token)
+        scoped_user_session = project_scoped_session(
+            project_id=ks_federated_project.id,
+            unscoped_token=unscoped_session.get_token(),
+            region=region)
+
+        admin_cinder = cinder_client.Client(session=sess)
+        user_cinder = cinder_client.Client(session=scoped_user_session)
+
+        volumes_to_migrate = [v for v in admin_cinder.volumes.list(
+            search_opts={
+                'all_tenants': 1,
+                'project_id': ks_legacy_project.id
+            }
+        )]
+
+        num_volumes = len(volumes_to_migrate) 
+        yield progress, f'There are {num_volumes} volumes to process.'
+
+        for index, volume in enumerate(volumes_to_migrate):
+
+            if volume.status == 'available':
+                transfer = admin_cinder.transfers.create(volume.id)
+                user_cinder.transfers.accept(transfer.id, transfer.auth_key)
+            else:
+                yield progress, (
+                    f'Volume {volume.name} is not available to transfer. '
+                    f'Please detach volume from any instances and re-run '
+                    f'migration.'
+                )
+
+            yield progress, f'Processed {index + 1} of {num_volumes} volumes.'
```

Comment:
`progress` is never incremented--incrementing progress is how the progress bar moves in the UI. Incrementing progress is however tricky w/ the current code, which assumes the only thing being migrated are disk images, and so when 100% of the disk images are migrated, the progress is 100% (1.0).

Reply:
Yea, I can fix this up.

Reply:
Updated the code. Looks to be working properly.